PixelHunter1995/Camera.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.Xna.Framework;
+using PixelHunter1995.Utilities;
+
+namespace PixelHunter1995
+{
+    class Camera
+    {
+        private readonly double CAMERA_SPEED = 1.5;
+        private readonly int START_MOVING_OFFSET = 100;
+        private readonly int STOP_MOVING_OFFSET = 2;
```

Comment:
I usually make these static, not that we will have more than one Camera though I guess...

Reply:
Fixed